### PR TITLE
Updates Readme and doctrisngs

### PR DESCRIPTION
**Pull request by gregordecristoforo**
The Readme section now contains a "why xarray?" section. I am not sure whether one should add even more (for instance code examples) since the Readme file might get to bloated. 

Doctrings are also updated and contain more explanations.

Lastly, loading input file variables is now done directly with json.laod()

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,15 @@ dataset.
...
 
+## Why xarray?
+
+xarray is a powerful open-source python library which aims to provide Pandas-like labelling, visualisation & analysis functionality for N-dimensional data. Some of the basic features are:
+* **Labelled multidimensional data**: xarray wraps numpy arrays with their `dims` as `DataArray`s.
+* **Clear syntax for operations**: xarray provides clearer and more general syntax containing less magic numbers than numpy 
+* **Lazy loading into memory**: never wastes RAM on unneeded values
+* **Plotting convenience**: xarray provides plotting functions (wrapping matplotlib) which automatically use an appropriate type of plot for the dimension of the data (1D, 2D etc.)
+
+For more information see the xarray [getting started guide](http://xarray.pydata.org/en/stable/getting-started-guide/why-xarray.html). Tm Nicholas' [presentation](https://github.com/TomNicholas/xBOUT--BOUT-workshop2019) about xarray and xBOUT is also a good place to start.
```
Comment (mwiesenberger) [inline README.md]: Tm -> Tom

File: README.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ dataset.
 
 xFELTOR is inspired by [xBOUT](https://github.com/boutproject/xBOUT) and uses currently some of its plotting functionality.
 
+## Why xarray?
+
+xarray is a powerful open-source python library which aims to provide Pandas-like labelling, visualisation & analysis functionality for N-dimensional data. Some of the basic features are:
```
Comment (mwiesenberger) [inline README.md]: evtl hier auch schon den link zu xarray einfügen [xarray]( ...)

File: xfeltor/load.py [hunk 3]
```diff
@@ -10,14 +10,19 @@ def open_feltordataset(
...
     datapath : str or (list or tuple of xr.Dataset), optional
         Path to the data to open. Can point to either a set of one or more *nc
         files.
     chunks : dict, optional
+        Dictionary with keys given by dimension names and values given by chunk sizes.
+        By default, chunks will be chosen to load entire input files into memory at once.
+        This has a major impact on performance: please see the full documentation for more details:
+        http://xarray.pydata.org/en/stable/user-guide/dask.html#chunking-and-performance
     restart_indices: bool, optional
         if True, dublicate time steps from restared runs are kept
```
Comment (mwiesenberger) [inline xfeltor/load.py]: dublicate -> duplicate

Comment (gregordecristoforo) [reply xfeltor/load.py]: Fixed in new commit

File: README.md (hunk 2 above)
Comment (gregordecristoforo) [reply README.md]: fixed in new commit

File: README.md (hunk 1 above)
Comment (gregordecristoforo) [reply README.md]: fixed in new commit

**Pull request merged**